### DISCUSSION TITLE
Unable to run npm build in Azure Pipeline

### DISCUSSION THREAD

**Discussion by jmriyaz84** [quasarframework/quasar#11073] [CLI - PWA mode] [answered]
[error_log.txt](https://github.com/quasarframework/quasar/files/7397529/error_log.txt)

<img width="762" alt="pipeline_issues" src="https://user-images.githubusercontent.com/8514576/138463317-85d16bf7-e8fa-47fb-b9a1-e2b5f710b005.PNG">
We have developed Quasar based project , which is running file in manual mode , we want to automate the build process to Azure  , so we have created the pipeline but when we rum the pipeline we are getting following error **" Field 'browser' doesn't contain a valid alias configuration"** , but if we run the same in Linux server it is working fine.

Please find Screenshot of error message & error log in the attachment.

Please help me to fix this.

**Comment by MilosPaunovic** [comment]
Have you tried to resolve the first error message:
`Module not found: Error: Can't resolve 'src/pages/crucibletracking/CMF.vue' in '/home/vsts/work/1/s/src/router'`

Are you importing that file from the wrong path or something similar?

**Reply by jmriyaz84** [reply]
But the same command is working when I run in my windows local machine.

**Reply by MilosPaunovic** [reply]
Since the action runner is Linux, maybe it's worth checking are the environment variables loading properly, because you're testing it in Windows local machine.

**Reply by rstoenescu** [reply]
> But the same command is working when I run in my windows local machine.

Linux has a case sensitive filesystem. Windows does not. Make sure that you use the correct casing on your imported file.

**Reply by jmriyaz84** [reply]
@rstoenescu Thanks for your quick help , yes it was a case sensitive issue it should be **crucibleTracking** instead of crucibletracking , now it is working .

Thanks a lot :)

**Comment by jmriyaz84** [comment]
Issue got resolved

**Comment by rstoenescu** [comment, marked as answer]
Linux has a case sensitive filesystem. Windows does not. Make sure that you use the correct casing on your imported file.
